brd-ios/breadwallet/src/Redesign/Scenes/Swap/Main/SwapInteractor.swift

```diff
@@ -196,19 +204,24 @@ class SwapInteractor: NSObject, Interactor, SwapViewActions {
         estimateFee(amount: to, currencyCode: dataStore?.toCurrency?.code) { [weak self] result in
             switch result {
             case .successFee(let fee):
+                self?.dataStore?.toFeeEth = nil
                 self?.dataStore?.toFee = fee
                 
             case .successEthFee(let ethFee):
+                self?.dataStore?.toFee = nil
                 self?.dataStore?.toFeeEth = ethFee
                 
             case .failure(let error):
+                self?.dataStore?.toFee = nil
+                self?.dataStore?.toFeeEth = nil
                 self?.presenter?.presentError(actionResponse: .init(error: error))
             }
             group.leave()
         }
         
         group.notify(queue: .main) { [weak self] in
             self?.calculateAmounts(viewAction: viewAction)
+            LoadingView.hide()
```

Comment:
viewcontroller stuff

Reply:
yp.. this was a drift fix.. cause the resposnes were so slooow i didnt rly know if i got em or not